README.md

```diff
@@ -1,159 +1,173 @@
-# Welcome\! 
-#### This repository contains the source code for:
+# Welcome
 
-  * [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
-  * The Windows console host (`conhost.exe`)
-  * Components shared between the two projects
-  * [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
-  * [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
-  
-#### Other related repositories include:
-  * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
+This repository contains the source code for:
 
-## Installation
+* [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
+* The Windows console host (`conhost.exe`)
+* Components shared between the two projects
+* [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
+* [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
 
-_(Note: in order to run the Windows Terminal, you'll need to be running at least Windows build 18362 or higher.)_
+Related repositories include:
 
-### Microsoft Store
+* [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
```

Comment:
Do we want to add the Cascadia Code repo here?

Reply:
Added. Thanks.